controllers/metal3cluster_controller.go

```diff
@@ -151,7 +150,7 @@ func patchMetal3Cluster(ctx context.Context, patchHelper *patch.Helper, metal3Cl
 	return patchHelper.Patch(ctx, metal3Cluster, options...)
 }
 
-func reconcileNormal(ctx context.Context, clusterMgr baremetal.ClusterManagerInterface) (ctrl.Result, error) {
+func reconcileNormal(ctx context.Context, clusterMgr baremetal.ClusterManagerInterface) (ctrl.Result, error) { //nolint:unparam
```

Comment:
Is this because the ctrl.Result is always empty?

Reply:
Precisely, I was tempted to remove it in the first glance, then thought otherwise since this is reconcile function, its better leave option for future user cases.